src/view3d.cpp

```diff
@@ -226,8 +226,14 @@ std::unique_ptr<QGLFramebufferObject> View3D::allocIncrementalFramebuffer(int w,
     fboFmt.setAttachment(QGLFramebufferObject::Depth);
     fboFmt.setSamples(fmt.samples());
     //fboFmt.setTextureTarget();
-    return std::unique_ptr<QGLFramebufferObject>(
-        new QGLFramebufferObject(w, h, fboFmt));
+    QGLFramebufferObject* fbo = new QGLFramebufferObject(w, h, fboFmt);
```

Comment:
Nitpick: I generally  prefer to wrap in `unique_ptr` immediately at allocation time to make early function exits safe from memory leaks (including via unexpected exceptions).